src/System.Private.CoreLib/shared/System/Number.Formatting.cs

```diff
@@ -1205,10 +1205,7 @@ private static unsafe void UInt32ToNumber(uint value, ref NumberBuffer number)
         {
             while (--digits >= 0 || value != 0)
             {
-                // TODO https://github.com/dotnet/coreclr/issues/3439
-                uint newValue = value / 10;
-                *(--bufferEnd) = (byte)(value - (newValue * 10) + '0');
-                value = newValue;
+                *(--bufferEnd) = (byte)(Math.DivRem(value, 10, out value) + '0');
```

Comment:
This is backwards. The result of `DivRem` is the division but you're using it like the remainder. I'm not sure why this change is needed to begin with.

Reply:
Oh, indeed, perhaps we should rename `uint DivRem(uint a, uint b, out uint result)` to `uint DivRem(uint a, uint b, out uint remainder)` ? It was a cosmetic change (to hide div hacks in one place)

Reply:
Renaming would be great but unfortunately it's a breaking change.

Reply:
I thought it's internal (unsigned overloads actually are) so yeah then, a breaking change.